qserver/qs_host.sh

```diff
@@ -0,0 +1,234 @@
+#!/bin/bash
+# file: qs_host.sh
+# Manage the bluesky queueserver host process.
+# Could be in a screen session or run as a direct process.
+
+SHELL_SCRIPT_NAME=${BASH_SOURCE:-${0}}
+SCRIPT_DIR="$( cd -- "$( dirname -- "${SHELL_SCRIPT_NAME}" )" &> /dev/null && pwd )"
+# CONFIGS_DIR="${SCRIPT_DIR}"/../src/instrument/configs
+command="import pathlib, instrument;"
+command+="print(pathlib.Path(instrument.__file__).parent / 'configs')"
+CONFIGS_DIR=$(python -c "${command}")
+
+###-----------------------------
+### Change program defaults here
+
+# Instrument configuration YAML file with databroker catalog name.
+ICONFIG_YML="${CONFIGS_DIR}"/iconfig.yml
+
+# Bluesky queueserver configuration YAML file.
+# This file contains the definition of 'redis_addr'.  (default: localhost:6379)
+QS_CONFIG_YML="${SCRIPT_DIR}/qs-config.yml"
+
+# Host name (from $hostname) where the queueserver host process runs.
+# QS_HOSTNAME=amber.xray.aps.anl.gov  # if a specific host is required
```

Comment:
For XPCS, it is required that this process only be started from workstation `amber`.  Reverse the commenting of these two lines.

This is likely a configurable parameter for the copier template.  (At least for XPCS, both redis & QS host processes are run on the same workstation).

Reply:
Later, these lines are a general enforcement of that requirement:
![image](https://github.com/user-attachments/assets/a8c871d9-ac82-4bf4-940d-92eeb01f4755)